tests/conftest.py

```diff
@@ -25,6 +25,16 @@
 from tools.local_http_server import LocalHttpServer
 
 
+@pytest_asyncio.fixture
```

Comment:
defining the same fixture twice

Reply:
Indeed. I screw up with rebasing... Fixed.